BaseFilter/sections/antiadblock.txt

```diff
@@ -1090,6 +1090,8 @@ pornhub.com,pornhub.org,pornhub.net#%#//scriptlet('prevent-setTimeout', '\x41\x6
 ! NOTE: Regular rules
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/180577
+donghuaworld.com#%#//scriptlet('abort-on-property-read', 'adsBlocked')

```

Comment:
We have a common rule for this. Add the domain to the rule in that file - you can find it under the comment.
```
! #%#//scriptlet('abort-on-property-read', 'adsBlocked')
```

Reply:
Also create a base specific rule for content blockers. Smth like this.
```adblock
donghuaworld.com###footer + div[id][class*=" "]
```
You can add it here.
<details><summary>Screenshot:</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/33488229/c2b39a7e-7365-491c-a50b-b0d342f0d27f)

</details><br/>